RELEASING.md

```diff
@@ -0,0 +1,20 @@
+# Release protocol
+This document describes the steps that need to be taken to create a new release for VAST.
+
+- Verify that the documentation in the `README.md`, the man page (`docs/man.1.md`) and https://github.com/vast-io/manual are up to date.
```

Comment:
Maybe provide some examples, of what "up to date" means? I can think of the following:
- Verify that all shown command lines properly execute. Actually, can we add the commands in the README to our integration tests? Then the task becomes making sure that there exist integration tests for the shown command lines.
- Make sure that the dependencies and their versions are correct.

We could remove the reference to the manual, because we really should integrate it into the main repository soon.

Reply:
> Verify that all shown command lines properly execute. Actually, can we add the commands in the README to our integration tests? Then the task becomes making sure that there exist integration tests for the shown command lines.
Yes, we can do that.

> - Make sure that the dependencies and their versions are `correct.
Good that you mention that. AFAIU we depend on unreleased versions of CAF and broker. Maybe its time to add the ability to automatically fetch those to the build system.

> We could remove the reference to the manual, because we really should integrate it into the main repository soon.
ok

Reply:
> we depend on unreleased versions of CAF and broker.

I think VAST releases should always depend on stable CAF/Broker versions. `master` can rely on CAF `master`, but as soon as we make a release, we should have a stable dependency.